ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/PartyManagementClient.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.ledger.client.services.admin
+
+import com.digitalasset.daml.lf.data.Ref.Party
```

Comment:
I see. I defined the `PartyManagementClient` in terms of `com.digitalasset.ledger.api.domain` types, I will change it to use the generated scalapb types.

By the way, if we don't want to use domain types, should we not change all usages `com.digitalasset.ledger.api.domain.LedgerId` to `String`?

Reply:
Yes, that's a fair point. Also, this should really be @leo-da's and @S11001001's call.

Reply:
`Ref.Party` is extremely convenient to use; if it is the intended meaning you should use it, because it is just a `String` where we've checked that only legal Party characters have been used.

I don't consider our use of `String` for Party in any Scala code to be principled, just that the good work that @remyhaemmerle-da has done tightening up the String subtypes in Engine and elsewhere hasn't propagated here yet.

As for domain types in general, @leo-da and I intend to push the Scala ecosystem _away_ from expressing _anything_ in terms of scalapb types, just as the Engine deals with values and transactions in terms of the `daml-lf/transaction` hand-written ADTs instead of the protoc output. In particular, we very much intend to universally remove the scalapb `Identifier`, and the transitive closure of all scalapb types that use it. So take that prejudice as you will.

In this case it seems preferable to use the domain types instead of scalapb as @rautenrieth-da has done. But _notwithstanding the above_ you may consider whether to use `Ref.Party` and whether to also use other domain types separately.

Reply:
Thanks for the explanation and outlook, @S11001001 👍